tools/fmt

```diff
@@ -42,7 +44,7 @@ fi
 
 prettier-eslint \
     --write \
-    --eslint-config-path ./tools/formatting.eslintrc.yaml \
+    --eslint-config-path $root_dir/tools/formatting.eslintrc.yaml \
```

Comment:
This is a good fix that seems like it can be in a separate prep commit. (Ditto in `tools/test` below.)

Reply:
Mm, thanks for catching these!